ads/google/a4a/test/test-line-delimited-response-handler.js

```diff
@@ -168,7 +168,9 @@ describe('#line-delimited-response-handler', () => {
       sandbox.restore();
     });
 
-    it('should handle empty streamed response properly', () => {
+    // TODO(amphtml, #15748): Fails on Safari 11.1.0.
```

Comment:
Can we change this to the a4a owner, @ mention them here and file/assign issues?

Reply:
/to @lannka (based on go/amp-components)